PubnativeLite/Scripts/generate.sh

```diff
@@ -31,4 +31,4 @@ zip -r $IPHONEOS_ZIP_PATH $IPHONEOS_FRAMEWORK
 zip -r $IPHONESIMULATOR_ZIP_PATH $IPHONESIMULATOR_FRAMEWORK
 
 # Generate Static framework + bundle resrource. a zip file will be generated at /tmp/circle-ci-artifact
-xcodebuild -workspace HyBid.xcworkspace -scheme HybidFramework -sdk iphonesimulator -configuration Release clean build | xcpretty -c
+xcodebuild -workspace HyBid.xcworkspace -scheme HybidFramework -sdk iphoneos -configuration Release clean build | xcpretty -c
```

Comment:
shouldn't we build for both?

Reply:
I have another run script in the Static framework target to build for both architectures and then merge. This is just to trigger the build and will take the iphoneos as the main architecture.

Reply:
Cool 👍